dockerfiles/compilation/java-8/integration_tests.sh

```diff
@@ -53,5 +53,5 @@ git clone $ORIGIN/.
 git checkout $SHA1
 
 
-mvn -DskipTests -pl org.apache.james:apache-james-mpt-external-james -am compile
+mvn -DskipTests -pl org.apache.james:apache-james-mpt-external-james -am clean install
```

Comment:
coupled with maven repository VOLUME you now have a stateful build system.
It's not a good idea _at all_.

Reply:
BTW, using a maven repository VOLUME is stateful...

Reply:
Yes, it's a cache of maven central, I don't have any problem with that.
But I have a problem using this VOLUME to store our artifacts, it will lead to old/missing artifacts being picked up when it shouldn't.

Reply:
Ok, I found a way to do both `mvn` commands at once.